runtime-multimedia/libva-nvidia-driver/spec

```diff
@@ -1,5 +1,4 @@
-VER=0.0.6
-REL=1
-SRCS="tbl::https://github.com/elFarto/nvidia-vaapi-driver/archive/refs/tags/v0.0.6.tar.gz"
-CHKSUMS="sha256::8a64c069200750f12d013c79547e459cbb87b498357478c5f36cb97f710294e0"
+VER=0.0.10
+SRCS="tbl::https://github.com/elFarto/nvidia-vaapi-driver/archive/refs/tags/v$VER.tar.gz"
```

Comment:
Please try not to use automatically packaged tarballs, use Git tags instead.

Reply:
Fixed.